crates/zome_types/src/metadata.rs

```diff
@@ -36,14 +32,14 @@ pub struct EntryDetails {
     pub entry: Entry,
     /// Create relationships.
```

Comment:
Maybe this code comment could be updated with the improved description you gave me in the chat, of what these fields represent?